web/locales/cs.yml

```diff
@@ -0,0 +1,6 @@
+cs:
+  RecurringJobs: Pravidelně opakované
+  Name: Jméno
```

Comment:
Also if they remove the Name translation from his files, we will not have that translation anymore.